content/_landing/index._zh.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: landing
+permalink: /zh/
+one_account_banner: true
+title: 公众进入政府网站只要一个账户
+description: 使用一个账号和密码来安全、私密地访问参与这个项目的政府机构。
+class: why-login-gov
+three_col:
+  heading: Login.gov 是为你服务的
+  subheading1: 个人
+  col_class1: individuals
+  col1: >-
+    使用一个账号和密码来安全、私密地访问参与这个项目的政府机构
+  
+
+    [了解 Login.gov](/zh/what-is-login/){:class="why-more-info"}
+  subheading2: 机构合作伙伴
+  col_class2: partners
+  col2: >-
+    用数字安全和用户体验的最高标准来保护你的用户信息Login.gov 管理软件开发、安全运营、以及客户支持，使你没必要做这些事情。
+  
+    [成为合作伙伴](/partners/){:class="why-more-info"}
+  subheading3: 机构开发商
+  col_class3: developers
+  col3: >-
+    程序员资源、实时支持、以及现代化工具来帮助你执行和部署自己在 Login.gov 的应用程序,
+
+
+    [参见开发人员指南]((https://developers.login.gov/){:class="why-more-info"}
```

Comment:
Looks like a syntax error here is preventing this from being rendered as a link.

```suggestion
    [参见开发人员指南](https://developers.login.gov/){:class="why-more-info"}
```

![image](https://github.com/GSA-TTS/identity-site/assets/1779930/5d375528-e3e1-4d58-86a4-5727ed8f8926)